### PR TITLE
Discuss proof stubs in conventions

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This was inspired by #4088.

### CODE REVIEW

**Comment by tirix** [comment]
@metakunt listed the axioms which we actually used as proof stubs until now in #4088.
These are:
- [ax-hgprmladder](https://us.metamath.org/mpeuni/ax-hgprmladder.html)
- [ax-bgbltosilva](https://us.metamath.org/mpeuni/ax-bgbltosilva.html)
- [ax-tgoldbachgt](https://us.metamath.org/mpeuni/ax-tgoldbachgt.html)
- [ax-hgt749](https://us.metamath.org/mpeuni/ax-hgt749.html)
- [ax-ros335](https://us.metamath.org/mpeuni/ax-ros335.html)
- [ax-ros336](https://us.metamath.org/mpeuni/ax-ros336.html)

They all have a mention like _Temporarily provided as axiom._ Maybe we could officialize that?

**Review by jkingdon — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
> They all have a mention like Temporarily provided as axiom. Maybe we could officialize that?

Good idea. I'll add that.

File: set.mm [hunk 1]
```diff
@@ -427246,6 +427246,24 @@ should sometimes not be broken up (e.g., because
...
+       <li><p><b>Proof stubs.</b>
+       It's sometimes useful to record partial proof results, e.g.,
+       incomplete proofs or proofs that depend on something else not fully
+       proven.
+       Some systems (like Lean) support a "sorry" axiom, which lets you assert
+       anything is true, but this can quickly run into trouble, because
+       the Metamath tooling is smart and may end up using it
+       to prove everything.
+       You can put "?" proof step in proofs when something is not fully proven;
+       this is a standard part of Metamath.
```
[the remark this answers is not in the document]
Comment (david-a-wheeler) [reply set.mm:427258]: Ah! Thanks! Will fix.

[the remark this answers is not in the document]
Comment (david-a-wheeler) [reply set.mm:427258]: I have modified it, I think it's clearer now.

**Review by tirix — APPROVED** [review]

**Review by GinoGiotto — APPROVED** [review]
FYI, unknown steps generate errors in mmj2, metamath-knife and mmverify.py, while metamath-exe produces a warning which is not treated as an error in the pipeline, so the metamath-exe check passes: https://github.com/GinoGiotto/set.mm/actions/runs/10062554455/job/27815363552#step:11:21

**Review by icecream17 — APPROVED** [review]
ive usually commented out incomplete work, but I don't know if that's a convention or anything

**Comment by david-a-wheeler** [comment]
> FYI, unknown steps generate errors in mmj2, metamath-knife and mmverify.py, while metamath-exe produces a warning which is not treated as an error in the pipeline, so the metamath-exe check passes

The *set* of verifiers won't pass, so it won't be accepted.

> I've usually commented out incomplete work, but I don't know if that's a convention or anything

I don't know if that's really a convention, so I propose just accepting this text for now, and we can add more later.

**Comment by david-a-wheeler** [comment]
We have several approvals. Let's just merge what we have now, and we can add more later if desired.

**Pull request merged**